rebasehelper/build_log_hooks/files_build_log_hook.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import difflib
+import os
+import re
+
+import six
+
+from rebasehelper.build_log_hook import BaseBuildLogHook
+from rebasehelper.helpers.macro_helper import MacroHelper
+from rebasehelper.logger import logger
+
+
+class FilesBuildLogHook(BaseBuildLogHook):
+    NAME = 'files'
+    CATEGORIES = None
+
+    @classmethod
+    def get_name(cls):
+        return cls.NAME
+
+    @classmethod
+    def get_categories(cls):
+        return cls.CATEGORIES
+
+    @classmethod
+    def _parse_build_log(cls, log_path):
+        """Parses a build log.
+
+        Args:
+            log_path (str): Path to the RPM build log.
+
+        Returns:
+            tuple: The first element is the type of error that was found
+                in the log (missing or deleted). The second element is
+                a set of problematic files.
+
+        """
+        try:
+            with open(log_path, 'r') as build_log:
+                lines = build_log.read().splitlines()
+        except IOError:
+            logger.info('There was an error opening %s', log_path)
+            return None, None
+
+        error_type = None
+        files = set()
+        next_line_is_missing = False
+
+        for line in lines:
+            # The file on the line is in the SPEC file but not in sources. Example line:
+            # File not found: /root/rpmbuild/BUILDROOT/pello-0.1.1-1.fc27.x86_64/should/not/be/here
+            match_deleted = re.match(r'(BUILDSTDERR:)?\s*File not found: (/.*)', line)
```

Comment:
So, what about this?
```py
re.compile(r'^(BUILDSTDERR:)?\s*((?P<missing>File not found:\s*)|(?P<unpackaged>Installed \(but unpackaged\) file\(s\) found:))?(?P<path>/.*)?$')
```

Single regexp for all cases, of course compiled before entering the loop.